pytorch/training/docker/1.10/py3/Dockerfile.e3.cpu

```diff
@@ -72,6 +72,27 @@ ENV PATH="$PATH:/home/.openmpi/bin"
 ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/home/.openmpi/lib/"
 RUN ompi_info --parsable --all | grep mpi_built_with_cuda_support:value
 
+# Install OpenSSH for MPI to communicate between containers, allow OpenSSH to talk to containers without asking for confirmation
+RUN apt-get update \
+ && apt-get install -y  --allow-downgrades --allow-change-held-packages --no-install-recommends \
```

Comment:
is this line required? Seems redundant

Reply:
I directly use the install commands in GPU training dockerfile [here](https://github.com/aws/deep-learning-containers/blob/master/pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu#L287-L305). prefer to leave it as is to avoid unexpected behaviors.

Reply:
This change seems to have originated here: https://github.com/aws/deep-learning-containers/pull/356#discussion_r461830982

I recommend trying to run one execution of tests to see whether it works without this line, because it sees like it should be completely redundant.

Reply:
rerun the build and test for e3 cpu training without that line. tests pass